hackathon_site/dashboard/frontend/src/testing/mockData.tsx

```diff
@@ -842,6 +842,8 @@ export const mockPendingOrdersInTable: OrderInTable[] = [
             },
         ],
         status: "Ready for Pickup",
+        createdTime: "Created at: 2021-10-17 18:28\xa0\xa0\xa0\xa0\xa0",
```

Comment:
Ideally we don't want to have to use xa0\xa0 and so on, the styling should handle the spacing